package.json

```diff
@@ -89,6 +89,7 @@
     "@vue/eslint-config-standard": "^5.1.2",
     "@vue/test-utils": "^1.1.0",
     "babel-eslint": "^10.1.0",
+    "bitcoin-address-generator": "^1.0.2",
```

Comment:
Can we avoid adding a new module for this one? From a security perspective, it's best to keep dependencies as low as possible

Reply:
I think it should be possible to use just `bitcoinjs-lib` which is already included in the wallet

Reply:
I found an example: https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/addresses.spec.ts#L13

Reply:
Removed that dependencies and used existing one